### ISSUE TITLE
Full  object previews

### ISSUE THREAD

**Issue by dbanksdesign** [aws-amplify/amplify-ui#6224]
### On which framework/platform would you like to see this feature implemented?

React

### Which UI component is this feature-request for?

Storage

### Please describe your feature-request in detail.

Full object previews for Storage Browser

### We love contributors! Is this something you'd be interested in working on?

- [ ] 👋 I may be able to implement this feature request.
- [ ] ⚠️ This feature might incur a breaking change.

**Comment by koiker** [comment]
It is not clear what the feature is.
If you download the full object, it will not become a preview anymore. It will become a copy of the object downloaded in the browser.
Can you articulate what exactly is the feature request?

**Comment by Sanjana1304** [comment]
Does Storage Browser provide preview file feature even without downloading it? @koiker

**Comment by koiker** [comment]
No, this feature is not part of StorageBrowser. But can be implemented by processing the rendering of items and prefetch the objects to preview. The component is very modular and allows you to interact with multiple steps of listing, downloading and uploading objects.
